### DISCUSSION TITLE
How to change the position of Row Actions in the table

### DISCUSSION THREAD

**Discussion by joaonivaldo** [filamentphp/filament#11785] [Help]
### Package

Table builder

### Package Version

v2.17.55

### How can we help you?

Hello everybody. Guys, I need to change the position of the Row Actions to the beginning of the line instead of the end. I tried using the "getTableActionsPosition" method, setting it as "Position::BeforeColumns", but the actions continue at the end of the line. Does anyone know how to resolve this? 
I've actually tried several things, but nothing makes the actions go to the beginning of the line. Remembering that I am using it on a Livewire component page and not on a Filament resourcePage. I already rebuild the assets and nothing either.
I tried with this code that is in the documentation and it didn't work.

protected function getTableActionsPosition(): ?string
{
    return Position::BeforeColumns;
}

**Comment by MasoodRehman** [comment]
You can do it by overriding the `makeTable` method of `InteractsWithTable` trait if you are using a table in a widget or page component.

## Widget or Page Component
```php 
    protected function makeTable(): Table
    {
        return parent::makeTable()
            ->actionsPosition(Tables\Enums\ActionsPosition::BeforeColumns);
    }
```

## In Resource Component:
<img width="581" alt="Screenshot 2024-07-22 at 2 40 08 PM" src="https://github.com/user-attachments/assets/24fbc6f3-c817-4892-b98d-b8b8349fe3b6">

## Result:

<img width="308" alt="Screenshot 2024-07-12 at 10 36 21 PM" src="https://github.com/user-attachments/assets/a68bac7c-48ae-4672-ae4e-b698e7eaf252">
